src/main/resources/META-INF/plugin.xml

```diff
@@ -49,6 +49,12 @@
                 <override-text place="GitMacheteContextMenu"/>
             </action>
 
+            <action id="GitMachete.SquashSelectedAction"
+                    class="com.virtuslab.gitmachete.frontend.actions.contextmenu.SquashSelectedAction"
+                    icon="MacheteIcons.SQUASH">
```

Comment:
Hmm when I hover over this action... I'm getting an incorrect status:

![image](https://user-images.githubusercontent.com/3383210/185400164-362f08ea-193d-403d-bf61-c3fe3fb2c097.png)

Reply:
Sum Ting Wong